### PR TITLE
[fix](meta) do not serialize nametotable map

### PR DESCRIPTION
**Pull request by dataroaring**
Gson can not handle size beyonds 2GB.

## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java [hunk 1]
```diff
@@ -650,6 +676,10 @@ public String getSignature(int signatureVersion) {
     public void write(DataOutput out) throws IOException {
         discardHudiTable();
         Text.writeString(out, GsonUtils.GSON.toJson(this));
+        if (FeMetaVersion.VERSION_138 <= Env.getCurrentEnvJournalVersion()) {
+            writeTables(out);
+            return;
+        }
```
Comment (w41ter) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:682]:
```suggestion
        writeTables(out);
        return;
```

**Comment by dataroaring** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java [hunk 2]
```diff
@@ -650,6 +676,8 @@ public String getSignature(int signatureVersion) {
     public void write(DataOutput out) throws IOException {
         discardHudiTable();
         Text.writeString(out, GsonUtils.GSON.toJson(this));
+        writeTables(out);
+        return;
```
Comment (w41ter) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:680]:
```suggestion
```

**Review by w41ter — DISMISSED** [review]

**Comment by dataroaring** [comment]
run buildall

**Review by w41ter — APPROVED** [review]

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**